cross/duplicity/patches/setup.py.patch

```diff
@@ -0,0 +1,63 @@
+--- setup.py.org	2014-05-09 15:27:41.000000000 +0200
++++ setup.py	2014-10-02 07:28:43.000000000 +0200
+@@ -25,6 +25,7 @@
+ from setuptools.command.test import test
+ from setuptools.command.install import install
+ from setuptools.command.sdist import sdist
++from setuptools.command.install_scripts import install_scripts
+ 
+ version_string = "0.6.24"
+ 
+@@ -32,6 +33,17 @@
+     print "Sorry, duplicity requires version 2.4 or later of python"
+     sys.exit(1)
+ 
++if sys.version_info[0] >= 3:
```

Comment:
I don't see how this part of the patch is necessary, seeing as we're only interested in Python 2.7 support. Please verify this is needed.